package/MDAnalysis/analysis/rms.py

```diff
@@ -310,6 +309,20 @@ def process_selection(select):
     select['reference'] = asiterable(select['reference'])
     return select
 
+def weight_type_check(weights,atoms,selection):
```

Comment:
Add a docstring or define this as a private method `weight_type_check`. I would prefer the latter.